### DISCUSSION TITLE
Rich-Select: Add an example or reference to model-value-changed

### DISCUSSION THREAD

**Discussion by lpoulter** [ing-bank/lion#1486] [General]
I was confused for a while how to listen for a value change. I tried the native `@change` but that didn't work. I found out I need to use `@model-value-changed` by searching the slack archives. Related why does listener for a change event not work?

**Comment by tlouisse** [comment]
The recommended usage is to use @model-value-changed everywhere: it's a normalized, single source of truth event for all FormControls. The reason change works for other elements is because they have a native input/select/textarea underneath that fires the event.

Also see: 
- https://lion-web.netlify.app/docs/systems/form/model-value/
- https://github.com/ing-bank/lion/discussions/24

If you have tips about how we can improve docs, let us know ;) I'm still hoping we have time to write a set of best practices and tutorials for forms one day.
